pkg/session/sync_upgrade.go

```diff
@@ -95,6 +97,13 @@ func SyncNormalRunning(s sessionctx.Context) error {
 		logger.Warn("resume the job failed", zap.Error(e))
 	}
 
+	if mgr, _ := dist_store.GetTaskManager(); mgr != nil {
+		err := mgr.AdjustTaskOverflowConcurrency(context.Background(), s)
```

Comment:
Will the upgraded server have less than 16 CPU cores but not the server executing the function of `SyncNormalRunning`? In addition, this concurrency has not been written to the `tidb_background_subtask`, so it cannot be determined in `AdjustTaskOverflowConcurrency`.

Reply:
We assume that all TiDB servers have the same specs...

Because `tidb_background_subtask` does not have this field in v7.5, there is no problem.